crate/cli/src/actions/google/keypairs/create_keypairs.rs

```diff
@@ -77,68 +86,67 @@ struct KaclsKeyMetadata {
 }
 
 impl CreateKeypairsAction {
+    async fn wrap_key(
+        key_id: &str,
+        csekey_id: &str,
+        kms_rest_client: &KmsClient,
+    ) -> Result<String, CliError> {
+        let (wrapped_private_key_object, _) =
+            export_object(kms_rest_client, key_id, true, Some(csekey_id), false, None).await?;
+
+        Ok(general_purpose::STANDARD.encode(wrapped_private_key_object.key_block()?.key_bytes()?))
+    }
```

Comment:
Some documentation would be welcome :)

For example, which algorithm is used to wrap the key?

Plus, since you pass `true` to the `unwrap` argument, is really `wrapped_private_key_object` well-named? My understanding is that this function only perform an encoding of the key bytes. Is this the wrapping mentioned?

Reply:
good point ! unwrap should be set to `false`